iac/terraform/aws/web-client/variables.tf

```diff
@@ -0,0 +1,20 @@
+variable "environment" {
+  type        = string
+  description = "Deployment environment"
+}
+
+variable "bloodconnect_domain" {
+  description = "The domain name for the web client"
+  type        = string
+}
+
+variable "cloudfront_header_response_policy_id" {
+  description = "Cloudfront Managed response header policy id for CORS policy"
+  type        = string
+  default     = "5cc3b908-e619-4b99-88e5-2cf7f45965bd"
```

Comment:
where did we get this value, if this is sensitive information, it should be stored as github secret

Reply:
This isn't any sensitive information. AWS have some own response header policies. AWS have some Id's like this which indicates which headers are to include. 
https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/using-managed-response-headers-policies.html

Reply:
Please add a terraform comment with the link you provided

Reply:
terraform has data blocks for things like this, please use it

Reply:
can you share an example please?